front/app/containers/Admin/projects/project/ideas/NewIdeaButtonDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+// intl
+import { injectIntl } from 'utils/cl-intl';
+import messages from '../messages';
+import { InjectedIntlProps } from 'react-intl';
+
+// components
+import {
+  Box,
+  Dropdown,
+  DropdownListItem,
+  Text,
+  Icon,
+} from '@citizenlab/cl2-component-library';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { colors } from 'utils/styleUtils';
+import clHistory from 'utils/cl-router/history';
+
+// hooks & services
+import useLocalize from 'hooks/useLocalize';
+import { IProjectData } from 'services/projects';
+import { IPhaseData } from 'services/phases';
+
+// style
+import styled from 'styled-components';
+import { getMethodConfig } from 'utils/participationMethodUtils';
+
+interface Props {
+  phases: IPhaseData[] | null;
+  project: IProjectData;
+  showDropdown: any;
+}
+
+const StyledDropdown = styled(Dropdown)`
+  margin-top: 0px;
+`;
+
+const StyledText = styled(Text)`
+  color: white;
+  margin: auto;
+  padding: 8px;
+`;
+
+const NewIdeaButtonDropdown = ({
+  phases,
+  project,
+  showDropdown,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  const DropdownContent = () => {
+    const localize = useLocalize();
+    if (!isNilOrError(phases)) {
+      return (
+        <>
+          {phases.map((phase, i) => (
```

Comment:
I cannot tell from this PR, but which phases are included here? Could this array include phases in which the current user is not allowed to post? In that case, such phases should be excluded from the list.

Reply:
Since this is for handling the Admin (back office) new idea button, I believe they are allowed to post in all phases at any time?

Reply:
Also only ideation and budgeting phases will show up here, if that is what you're referring to :)

Reply:
> I believe they are allowed to post in all phases at any time?
Correct.